README.md

```diff
@@ -74,10 +74,11 @@ Elasticsearch is used when `STORAGE=elasticsearch`.
     * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication. Use when X-Pack security
                                        (formerly Shield) is in place. By default no username or
                                        password is provided to elasticsearch.
-    * `ES_CLIENT_NODE_ONLY`: Set to true to disable elasticsearch cluster nodes.discovery and enable nodes.client.only. 
-                             If your elasticsearch cluster's data nodes only listen on loopback ip, set this to true. 
+    * `ES_CLIENT_NODE_ONLY`: Set to true to disable elasticsearch cluster nodes.discovery and enable nodes.client.only.
+                             If your elasticsearch cluster's data nodes only listen on loopback ip, set this to true.
                              Defaults to false
     * `ES_INDEX_PREFIX`: index prefix of Jaeger indices. By default unset.
+    * `ES_SPAN_RANGE`: How far into the past should the job look, the maximum and default is 24h.
```

Comment:
Could you add a link to ES docs that lists what values can be used in this query?

Reply:
Could we rename this to `ES_TIME_RANGE`? And document that span start time is used for the comparison.

Reply:
Added, I struggled a bit with wording. How does the change sound?

Reply:
It looks good :+1: